src/bindings/js/node/lib/addon.ts

```diff
@@ -21,6 +21,12 @@ type elementTypeString =
   | 'f32'
   | 'string';
 
+type Any = string | number | boolean;
```

Comment:
I propose to choose different name, because `Any` crosses with `any` in TypeScript definition. 
It can be longer, but cleaner, for example: `AllowedPropertyType`, `ConfigPropertyType`, etc

Reply:
I updated the PR description and I think the best name will be OVAny